program/src/instruction.rs

```diff
@@ -463,55 +467,6 @@ accounts_struct! {
             is_writable: true,
         },
 
-        // We only allow updating balances if the exchange rate is up to date,
```

Comment:
Also here, we can’t modify existing instructions because that is a breaking change, we need to add _new_ instructions instead.

If it is possible to upgrade the old call in-place in the instruction dispatcher that is nice, but if that is not possible or not worth the effort, we can also make the old instruction fail. But we should not change the meaning of the existing instruction.